UPGRADE.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade
 
+## 1.7.5
```

Comment:
In terms of semver, deprecations should come in minor versions, not patch versions, is it a problem if you need to push these changes to 1.8.0 instead of 1.7.5?